package.json

```diff
@@ -30,6 +30,7 @@
     "@vue/cli-plugin-babel": "^4.5.10",
     "@vue/cli-plugin-eslint": "^5.0.0-alpha.2",
     "@vue/cli-service": "^4.5.10",
+    "cache-loader": "^4.1.0",
```

Comment:
Nothing in this PR appears to have needed us to pull in a new dependency?

Reply:
This might be to deal with the warning messages at start of runtime